examples/plot_wlasso.py

```diff
@@ -5,95 +5,112 @@
 
 This example shows how to perform hyperparameter optimization
 for a weighted Lasso using a held-out validation set.
-
+In particular we compare the weighted Lasso to LassoCV on a toy example
 """
 
 # Authors: Quentin Bertrand <quentin.bertrand@inria.fr>
 #          Quentin Klopfenstein <quentin.klopfenstein@u-bourgogne.fr>
-#
+#          Kenan Sehic
 # License: BSD (3-clause)
 
-import time
-
 import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
 
+from celer import Lasso, LassoCV
 from sklearn.datasets import make_regression
 from sklearn.model_selection import train_test_split
-
-from celer import Lasso
+from sklearn.metrics import mean_squared_error
 
 from sparse_ho.models import wLasso
 from sparse_ho.criterion import CV
 from sparse_ho.implicit_forward import ImplicitForward
 from sparse_ho.utils import Monitor
 from sparse_ho.ho import grad_search
-from sparse_ho.datasets import get_data
 
-print(__doc__)
+##############################################################################
+# Dataset creation
+##############################################################################
+X, y = make_regression(n_samples=600, n_features=600, noise=60, random_state=2)
```

Comment:
how do you choose this value for noise ?

Reply:
`noise=0` gives an error of 0 on the test and validation set.
I chose the noise for the problem to be interesting to solve.

Reply:
yes yes, but my point is that `noise=60` is not an "interpretable" value for me (I have no clue if it is high or not). I prefer parametrizing with SNR

Reply:
got it, I will change this